### PR TITLE
[HOPSWORKS-1021] Python requests - convert jks to pem

### PR DESCRIPTION
**Pull request by tkakantousis**

### CODE REVIEW

File: recipes/yarn.rb [hunk 1]
```diff
@@ -220,11 +220,21 @@
...
+  end
+
+  bash 'materialize_truststore and convert to pem' do
     user "root"
     code <<-EOH
         cp -f #{node['kagent']['certs_dir']}/cacerts.jks /tmp
         chmod 755 /tmp/cacerts.jks
+        keytool -importkeystore -srckeystore cacerts.jks -destkeystore cacerts.p12 -srcstoretype jks -deststoretype pkcs12 -noprompt -srcstorepass #{encyption_password} -deststorepass #{encyption_password} 
+        openssl pkcs12 -in cacerts.p12 -out cacerts.pem -passin pass:#{encyption_password}
+        history -c
```
Comment (kouzant) [inline recipes/yarn.rb]: Clearing the whole history might be too aggressive. Try `echo "YOUR_COMMAND && history -d $((HISTCMD-1))` instead

File: recipes/certs.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  code <<-EOH
+        su #{node['hops']['user']}
+        #{node['hops']['base_dir']}/bin/hdfs dfs copyToLocal "/user/#{node['hadoop_spark']['user']}/cacerts.jks" "#{node['hadoop_spark']['conf_dir']}/"
+        chmod 444 "#{node['hadoop_spark']['conf_dir']}/cacerts.jks"
+        #{node['hops']['base_dir']}/bin/hdfs dfs copyToLocal "/user/#{node['hadoop_spark']['user']}/cacerts.pem" "#{node['hadoop_spark']['conf_dir']}/" 
+        chmod 444 "#{node['hadoop_spark']['conf_dir']}/cacerts.pem"
+  EOH
+end
+
+hops_hdfs_directory "/user/#{node['hadoop_spark']['user']}/cacerts.jks" do
```
Comment (kouzant) [inline recipes/certs.rb]: When one node reaches this block, it will delete the files from HDFS. How will the rest of the machines localize them?

Comment (tkakantousis) [reply recipes/certs.rb]: @kouzant You are right! How would we clean up the file from hdfs then? We could also leave it there, doesn't hurt I guess

Comment (kouzant) [reply recipes/certs.rb]: There is no direct way. There has to be a recipe that must run after hadoop_spark::yarn has ran globally and delete the source files. Yes, we could leave them there as long as all `copyFromLocal` operations use `-f` flag to overwrite them.

**Pull request merged**